recipes/pixy/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "pixy" %}
+{% set version = "0.1" %}
+{% set sha256 = "c417e49e94cc8270c3c2595a4e7e8029e2b7142f1af3f0dd30ca460842ff31a7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ksamuk/pixy/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

You need to declare the entry points here

```suggestion
  number: 0
  entry_points:
    - pixy=pixy.py
```